### PR TITLE
Modified README

### PR DESCRIPTION
**Pull request by rahuldecoded**
Did some simple modifications and added a crucial step no 3. Without getting inside the folder with cannot use `git checkout`

### CODE REVIEW

**Comment by akram-rameez** [comment]
spelling mistakes bruh

**Comment by rahuldecoded** [comment]
@akram-rameez where is spelling mistakes ?

File: README.md [hunk 1]
```diff
@@ -4,22 +4,26 @@ Project Euler Answers
...
+1. [Fork this repository](https://github.com/akram-rameez/projecteuler.git) (Click the Fork button in the top right of this page, click your Profile Image)
+2. Clone your fork down to your local machine
 ```markdown
-git clone https://github.com/akram-rameez/projecteuler.git
+git clone https://github.com/<your_username>/projecteuler.git
 ```
-* Create a branch
+3. Get inside the `projecteuler` directory.
+```markdown
+cd projecteular
```
Comment (akram-rameez) [inline README.md:14]: spelling mistake

Comment (rahuldecoded) [reply README.md:14]: Which line ?

Comment (rahuldecoded) [reply README.md:14]: Okay got it

**Pull request closed without merging**